staging/src/k8s.io/client-go/examples/in-cluster-client-configuration/README.md

```diff
@@ -25,7 +25,26 @@ build the image on Minikube:
     docker build -t in-cluster .
 
 If you are not using Minikube, you should build this image and push it to a registry
-that your Kubernetes cluster can pull from.
+that your Kubernetes cluster can pull from. If you have RBAC enabled, use the following
+snippet to create role binding which will grant the default service account view
+permissions.
+
+```
```

Comment:
Could be a one-liner `kubectl create clusterrolebinding default-view --clusterrole=view --serviceaccount=namespace:namespace`, according to [doc](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#default-roles-and-role-bindings).

Reply:
I assume `view` is a default role that exists in every cluster, but couldn't really find documentation to support that.

Reply:
yes, it's a default role when RBAC is enabled. see https://kubernetes.io/docs/reference/access-authn-authz/rbac/#default-roles-and-role-bindings